Directory.Build.props

```diff
@@ -1,6 +1,7 @@
 ﻿<Project>
   <PropertyGroup>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TargetFramework>net5.0</TargetFramework>
+    <!-- <TreatWarningsAsErrors>true</TreatWarningsAsErrors> -->
```

Comment:
Is there a reason we are turning this off?

Reply:
This was on purpose for a step, but need to undo it, thanks for catching that